core/src/main/java/org/testcontainers/containers/DockerComposeContainer.java

```diff
@@ -465,6 +497,18 @@ private SELF self() {
     private String randomProjectId() {
         return identifier + Base58.randomString(6).toLowerCase();
     }
+
+    public enum RemoveImages {
+        /**
+         * Remove all images used by any service.
+         */
+        all,
```

Comment:
perhaps we should use standard naming convention for enums (upper case) and have a field with flag's value?

Reply:
Sure, can  do it.

Reply:
I've taken this on and am pushing the proposed change now.